src/index.js

```diff
@@ -0,0 +1,188 @@
+import syntaxJSX from "@babel/plugin-syntax-jsx";
+
+/**
+ * @param {import("@babel/core")} babelApi
```

Comment:
There are some of these JSDoc annotations to get editor suggestions, it's incredibly helpful when working with ASTs.

Reply:
It’s ok … this module is for Babel